Sources/Sentry/SentryHttpTransport.m

```diff
@@ -232,6 +232,7 @@ - (SentryFlushResult)flush:(NSTimeInterval)timeout
 - (void)envelopeItemDropped:(SentryEnvelopeItem *)envelopeItem
                withCategory:(SentryDataCategory)dataCategory;
 {
+    SENTRY_LOG_WARN(@"Envelope item dropped due to exceeding rate limit.");
```

Comment:
In case a pattern of `"RateLimit [error-description]"` has to be followed, that would be my alt suggestion: `RateLimit is active for current envelope item, dropping...`